firmware/thruna/PDM/Inc/App/App_PowerManager.h

```diff
@@ -0,0 +1,20 @@
+#include <stdbool.h>
+#include "Io_Efuse.h"
+
+typedef enum
+{
+    POWER_MANAGER_CONTRACTOR_OPEN,
+    POWER_MANAGER_CONTRACTOR_CLOSED,
```

Comment:
Maybe rename `POWER_MANAGER_CONTRACTOR_OPEN` -> `POWER_MANAGER_STATE_SHUTDOWN`, and -> `POWER_MANAGER_CONTRACTOR_CLOSED` -> `POWER_MANAGER_STATE_DRIVE`. This is because there will be a few seconds before we close contactors where we'll enable high power to turn on the inverters.